src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(19);
+        user.setLogin("evreii");
+        user.setPassword("helloMates");
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RuntimeException.class, () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        user.setAge(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        user.setLogin(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_nullPassword_notOk() {
+        user.setPassword(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_invalidAge_notOk() {
+        user.setAge(15);
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_invalidPassword_notOk() {
+        user.setPassword("12345");
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_existingLogin_notOk() {
+        user.setLogin("Evreii");
+        User expected = user;
+        User actual = registrationService.register(user);
+        assertEquals(expected, actual);
+    }
```

Comment:
id doesn't seem like not ok, you should check if register throws runtimeException, for this purpose you need to register you user twice